game-headed/build.gradle

```diff
@@ -41,7 +41,29 @@ jar {
     }
 }
 
-task downloadPlatformInstallerAssets(group: 'release') {
+// todo: have this run automatically on 'run' and or downloadPlatformInstallerAssets
```

Comment:
This TODO seems redundant

Reply:
Indeed, it was a very early placeholder and note to myself. I meant to remove it after finishing the task. Now removed: 0c3d44c33